pkg/service/wire_gen.go

```diff
@@ -89,23 +89,23 @@ func InitializeServer(conf *config.Config, currentNode routing.LocalNode) (*Live
 	}
 	rtcEgressLauncher := NewEgressLauncher(egressClient, ioInfoService)
 	topicFormatter := rpc.NewTopicFormatter()
-	v, err := rpc.NewTypedRoomClient(clientParams)
+	roomClient, err := rpc.NewTypedRoomClient(clientParams)
```

Comment:
not sure why `wire` is doing this, it switches the variable name on every run it seems like. Maybe, I need to update `wire` version. Will check.

Reply:
yeah likely a version difference.. try installing the latest version

Reply:
it seemed to have the latest already. Anyhow, installed again. Will keep an eye.